db/data_migrations/2022080530154709_fix_broken_comm_code_descriptions.rb

```diff
@@ -0,0 +1,8 @@
+Sequel.migration do
+  up do
+    Sequel::Model.db[:goods_nomenclature_descriptions_oplog].where(id: 154_585).update(description: 'Tunas (of the genus Thunnus), skipjack tuna (stripe-bellied bonito) (Katsuwonus pelamis), excluding edible fish offal of subheadings 0302 91 to 0302 99')
```

Comment:
I think you'll want the primary key (not labelled id) which is goods_nomenclature_description_period_sid.

Probably better to always use the table as configured by the model, too:

```ruby
description = GoodsNomenclatureDescription.where(goods_nomenclature_description_period_sid: 154_585)
description.update(description: 'Tunas (of the genus Thunnus), skipjack tuna (stripe-bellied bonito) (Katsuwonus pelamis), excluding edible fish offal of subheadings 0302 91 to 0302 99')
```

Reply:
If you really want to update the full history of descriptions that match that description sid, then you'll want to use the Operation class:

```ruby
description = GoodsNomenclatureDescription::Operation.where(goods_nomenclature_description_period_sid: 154_585)
description.update(description: 'Tunas (of the genus Thunnus), skipjack tuna (stripe-bellied bonito) (Katsuwonus pelamis), excluding edible fish offal of subheadings 0302 91 to 0302 99')
```

Reply:
Don't think we need the operation class for this purpose